legacy/src/api/assets/tokens.md

```diff
@@ -110,6 +113,8 @@ A redemption condition is created for each merchant that accepts tokens from a c
 	"status": "active",
 	"createdBy": "crn::user:b657195e-dc2f-11ea-8566-e7710d592c99",
 	"createdAt": "2021-05-12T04:30:11.001Z",
+	"mediaUploadId": "12345",
+	"img": "https://static.centrapay.com/assets/collections/{mediaUploadId}.png"
```

Comment:
Let's change this `img` URL to a more realistic one :)

Reply:
@Ray2126  something like this ?
https://static-assets-bucket-centrapay.s3.ap-southeast-1.amazonaws.com/{name}.png